src/lib/logger.js

```diff
@@ -46,42 +57,79 @@ function setup(logs) {
       return JSON.stringify({...obj, msg}, Logger.safeCycles()) + '\n';
     };
 
-    if (target.type === 'file') {
```

Comment:
From line 53 to 58, the following code is duplicated below and not being unused.
```
    const prettyPrint = (obj) => print(obj.level, obj.msg, obj, destIsTTY) + '\n';
    const prettyTimestampedPrint = (obj) => obj.time.toISOString() + print(obj.level, obj.msg, obj, destIsTTY) + '\n';
    const jsonPrint = (obj) => {
      const msg = fillInMsgTemplate(obj.msg, obj, destIsTTY);
      return JSON.stringify({...obj, msg}, Logger.safeCycles()) + '\n';
    };
```

Reply:
Meh, bad rebase on my part. Will fix